ingress/src/main/java/com/redhat/developer/ingress/producer/KafkaEventPublisher.java

```diff
@@ -0,0 +1,38 @@
+package com.redhat.developer.ingress.producer;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+import org.jboss.logging.Logger;
+import org.reactivestreams.Publisher;
+
+import com.redhat.developer.infra.utils.CloudEventUtils;
+import com.redhat.developer.ingress.api.exceptions.IngressRuntimeException;
+
+import io.cloudevents.CloudEvent;
+import io.smallrye.mutiny.operators.multi.processors.BroadcastProcessor;
+
+@ApplicationScoped
+public class KafkaEventPublisher {
+
+    private final Logger logger = Logger.getLogger(KafkaEventPublisher.class);
+
+    private final BroadcastProcessor<String> eventSubject = BroadcastProcessor.create();
+
+    public void sendEvent(CloudEvent cloudEvent) {
+        logger.info("Sending event to event queue");
```

Comment:
Might be useful to have some detail about the event we're sending in this log message to help us debug. Is there an `id` or something else than can help us uniquely identify the event we're sending?

Reply:
done